src/codec.ts

```diff
@@ -16,9 +16,9 @@
 import {Service} from '@google-cloud/common-grpc';
 import {DateStruct, PreciseDate} from '@google-cloud/precise-date';
 import * as arrify from 'arrify';
-import {CallOptions} from 'google-gax';
 import * as is from 'is';
 import {common as p} from 'protobufjs';
+import {google as spanner_client} from '../proto/spanner';
```

Comment:
Why the `spanner_client` alias? Up until now we've used `google` everywhere, plus we should always prefer camel casing.

Reply:
Hey! Just because there are two other proto files with the same exported `google` namespace, so  the `spanner_client` alias is to differentiate the `spanner.d.ts` file from `spanner_database_admin.d.ts` and `spanner_instance_admin.d.ts`